load_data/load_coverage.py

```diff
@@ -3,19 +3,26 @@
 
 
 def get_coverage_expr(mt):
+
+    cov_arrays = hl.literal({
```

Comment:
Any reason not to have 100 in here?

Reply:
since we stop binning at 100  I don't think it makes too much sense. Would you want something like
```
.when(mt.coverage >= 100, cov_arrays[-1])
```
?

Reply:
Fair enough. I guess it doesn't matter too much